### PR TITLE
Added Free Certified Course on Game Development by Michigan State University

### PR DESCRIPTION
**Pull request by arushmangal**
## What does this PR do?
Add resource

### Description
If you love games and want to learn how to make them, then this course will start you down that path. Making games is a creative and technical art form. In this course you will familiarize yourself with the tools and practices of game development and well as the process. You will get started developing video games using industry standard game development tools, including the Unity 2020 game engine. At the end of the course you will have completed two hands-on projects, including an Intro to Unity project and a 2D Shooter game, and will be able to leverage an array of game development techniques to create your own basic games.

### Why is this valuable (or not)?
The course extensively covers the entire working process of game development and helps beginners understand how to effectively use the Unity Game Development Service to create amazing games.

### How do we know it's really free?
You can easily go to the course on Coursera and click on 'Enroll for free', Audit this course' to access it free of cost

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by eshellman — CHANGES_REQUESTED** [review]
Is this course time limited? i.e. would this need to be removed in 6 months? If so, we shouldn't add it.

**Comment by arushmangal** [comment]
> Is this course time limited? i.e. would this need to be removed in 6 months? If so, we shouldn't add it.

No, it's not limited. Please do let me know what indicated the same to you incase I missed something

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -429,6 +430,7 @@
 * [Godot Action RPG Series](https://www.youtube.com/playlist?list=PL9FzW-m48fn2SlrW0KoLT4n5egNdX-W9a) - HeartBeast
 * [Godot Space Sidescroller Tutorial Series](https://www.youtube.com/playlist?list=PL6bQeQE-ybqAzXZlZCiRKCtu6RbkXLgmh) - PlugWorld
 * [Godot Wave Shooter Tutorials](https://www.youtube.com/playlist?list=PL6bQeQE-ybqAYoaWz_ZEE2X4wX6PhwCWR) - PlugWorld
+* [Low Poly Art For Video Games](https://www.coursera.org/learn/low-poly-art-video-games)
```
Comment (davorpa) [inline courses/free-courses-en.md:433]: Adding instructors

```suggestion
* [Low Poly Art For Video Games](https://www.coursera.org/learn/low-poly-art-video-games) - Andrew Dennis, Ricardo Guimaraes (Coursera)
```

Comment (arushmangal) [reply courses/free-courses-en.md:433]: Oops! Sorry, seems that I missed that!

**Comment by davorpa** [comment]
Duplicated of #7479

**Comment by davorpa** [comment]
Duplicate of ##7478

**Pull request closed without merging**